src/components/PersonCard/PersonCard.tsx

```diff
@@ -79,7 +79,12 @@ export const PersonCard: FC<PersonCardProps> = ({
   return (
     <Card className={classes.personCard}>
       <CardContent className={classes.cardContent}>
-        <Picture className={classes.personImage} source={imageSource} />
+        <Avatar
+          variant="circle"
+          alt={name}
+          src={imageSource}
+          className={classes.personImage}
+        />
```

Comment:
Generally I don't like "sneaking" in unrelated changes, but I'll let it fly this time 😄 
If you want to make other PRs unrelated to a ticket, be my guest!

Reply:
Will make pr's for unrelated change in future